### ISSUE TITLE
Nginx is unable to reach st2api

### ISSUE THREAD

**Issue by dbarentine** [StackStorm/st2-docker#66]
I'm currently trying to use the stackstorm/stackstorm:2.4.1 container with external mongo, rabbitmq, postgres, and redis servers. Everything seems to start up fine but when I try to log in I get an error: "Nginx is unable to reach st2api. Make sure service is running."

In the container I can see that st2api is running but netstat shows nothing is listening on http://127.0.0.1:9101. I've included the st2api log file below.

This one is the initial log file created after the container first started.
[st2api_after_container_start.log](https://github.com/StackStorm/st2-docker/files/1385785/st2api_after_container_start.log)

After that I enabled debug in st2.conf and deleted the log file and restarted st2api to get this log file.
[st2api_after_debug_enabled.log](https://github.com/StackStorm/st2-docker/files/1385786/st2api_after_debug_enabled.log)

There are some messages about duplicate keys but nothing that really stands out to me beyond that. But it looks like it might be in a loop trying to initialize.

**Comment by dbarentine** [comment]
I ran across this PR and comment: https://github.com/StackStorm/st2-docker/pull/15#issuecomment-299335207

When I remove the [coordination] url (i.e., redis) from st2.conf and restart st2api everything then seems to work fine. I tried running "/opt/stackstorm/st2/bin/pip install redis" inside the container (per the policies documentation) but it says the requirement is already satisfied. I verified my Redis connection to ensure everything was right there as well as well.
